Kommunicate/Classes/ConversationVCNavBar.swift

```diff
@@ -0,0 +1,161 @@
+    
+    let backButton: UIButton = {
+        let button = UIButton(type: .custom)
+        button.setImage(UIImage(named: "icon_back", in: Bundle.kommunicate, compatibleWith: nil), for: .normal)
+        button.isUserInteractionEnabled = true
+        return button
+    }()
+
+    var profileImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "placeholder", in: Bundle.kommunicate, compatibleWith: nil)
+        return imageView
+    }()
+    
+    var profileName: UILabel = {
+        let label = UILabel()
+        label.text = "No Name"
+        label.font = UIFont(name: "HelveticaNeue", size: 16) ?? UIFont.systemFont(ofSize: 16)
+        label.textColor = UIColor(red: 96, green: 94, blue: 94)
+        return label
+    }()
+    
+    lazy var statusIconBackgroundColor: UIView = {
+        let view = UIView()
+        view.backgroundColor = self.navigationBarBackgroundColor
+        return view
+    }()
+    
+    var onlineStatusIcon: UIView = {
+        let view = UIView()
+        view.backgroundColor = UIColor(165, green: 170, blue: 165)
+        return view
+    }()
+    
+    var onlineStatusText: UILabel = {
+        let label = UILabel()
+        label.text = "Offline"
+        label.font = UIFont(name: "HelveticaNeue", size: 12) ?? UIFont.systemFont(ofSize: 12)
+        label.textColor = UIColor(red: 113, green: 110, blue: 110)
+        return label
+    }()
+    
+    lazy var profileView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [self.profileName, self.onlineStatusText])
+        stackView.alignment = .fill
+        stackView.axis = .vertical
+        stackView.distribution = .fillEqually
+        stackView.spacing = 2
+        return stackView
+    }()
+    
+    
+    @objc func backButtonClicked(_ sender: UIButton) {
+        delegate.backButtonPressed()
+    }
+
+    
+    private func setupConstraints() {
+        self.addViewsForAutolayout(views: [backButton, profileImage, statusIconBackgroundColor, onlineStatusIcon, profileView])
+        
+        //Setup constraints
+        backButton.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 0).isActive = true
+        backButton.topAnchor.constraint(equalTo: self.topAnchor, constant: 10).isActive = true
+        backButton.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -10).isActive = true
+        backButton.widthAnchor.constraint(equalToConstant: 20).isActive = true
+
+        profileImage.leadingAnchor.constraint(equalTo: backButton.trailingAnchor, constant: 10).isActive = true
+        profileImage.topAnchor.constraint(equalTo: self.topAnchor, constant: 5).isActive = true
+        profileImage.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -5).isActive = true
+        profileImage.widthAnchor.constraint(equalToConstant: 35).isActive = true
+        profileImage.layer.cornerRadius = 18
+        profileImage.clipsToBounds = true
```

Comment:
check if we can add an alphabetic letter in case if image url is not there will show the first letter alphabetic